pydax/_dataset.py

```diff
@@ -161,46 +162,61 @@ def is_downloaded(self) -> bool:
         return self._data_dir.is_dir() and len(os.listdir(self._data_dir)) > 0
 
 
+@no_type_check
+def _handle_name_param(func):
```

Comment:
Just to add some readability:

```suggestion
def _handle_name_param(func: Callable) -> Callable:
```